test/e2e/storage/volume_provisioning.go

```diff
@@ -407,9 +408,7 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
 			}
 		})
 
-		// NOTE: Slow!  The test will wait up to 5 minutes (framework.ClaimProvisionTimeout)
-		// when there is no regression.
-		It("should not provision a volume in an unmanaged GCE zone. [Slow]", func() {
+		It("should not provision a volume in an unmanaged GCE zone.", func() {
```

Comment:
Since the max poll wait is 5m won't this test still be considered potentially slow?